homeworks/olya.vakulenko_Oya-Ol/4-objects/objects.js

```diff
@@ -0,0 +1,135 @@
+const tamagochi = {
+    name: 'Pikachu',
+    health: 10,
+    satiety: 10,
+    happiness: 10,
+    wealth: 10,
+    get alive() {
+        return this.health > 1 && this.satiety > 1 && this.happiness > 1 && this.wealth > 1;
+    },
+    // eslint-disable-next-line consistent-return
```

Comment:
Please, return `this` instead and remove all comments like this.

Reply:
done